Bobcat/Profile.cpp

```diff
@@ -12,21 +12,12 @@ namespace Upp {
 #define IMAGECLASS Images
 #include <Draw/iml_source.h>
 
-struct DefaultProfileNameDisplayCls : Display
+void ProfileNameDisplay::Paint(Draw& w, const Rect& r, const Value& q, Color ink, Color paper, dword style) const
 {
-	void Paint(Draw& w, const Rect& r, const Value& q, Color ink, Color paper, dword style) const final
-	{
-		StdDisplay().Paint(w, r, AttrText(q).SetImage(Images::Terminal()).Bold(), ink, paper, style);
-	}
-};
-
-struct NormalProfileNameDisplayCls : Display
-{
-	void Paint(Draw& w, const Rect& r, const Value& q, Color ink, Color paper, dword style) const final
-	{
-		StdDisplay().Paint(w, r, AttrText(q).SetImage(Images::Terminal()), ink, paper, style);
-	}
-};
+	const Image& img = ctx.settings.defaultprofile == q ? Images::DefaultTerminal() : Images::Terminal();
+	bool current = ctx.GetActiveTerminal()->profilename == q;

```

Comment:
Again, active terminal may not be present, GetActiveTerminal may return nullptr and then dialog will crash.

Reply:
Ditto...